Dockerfile

```diff
@@ -30,6 +30,14 @@ ENV ENABLE_WEBHOOKS=${ENABLE_WEBHOOKS}
 # See https://catalog.redhat.com/software/containers/ubi9/ubi-micro/615bdf943f6014fa45ae1b58
 FROM registry.access.redhat.com/ubi9/ubi-micro:9.1.0
 COPY --from=builder /opt/app-root/src/manager /
+
+# It is mandatory to set these labels
+LABEL \
+ description="RHTAP Release Service" \
+ io.k8s.description="RHTAP Release Service" \
+ io.k8s.display-name="release-service" \
+ summary="RHTAP Release Service"
```

Comment:
There's no benefit on declaring labels this way. You can use the LABEL keyword in each line without affecting the image size. But if you opt for this way of declaring it, could you do it like:
```
LABEL description="RHTAP Release Service" \
            io.k8s.description="RHTAP Release Service" \
            io.k8s.display-name="release-service" \
            summary="RHTAP Release Service"
```

Reply:
He started with your first paste and I asked him to change. Not sure why you want the `LABEL` instruction repeated

Reply:
Having said that, multiple LABEL keywords is nice for greping the file :wink:

Reply:
so revert to separate LABEL lines?

Reply:
Any of the examples I put is fine by me. Up to you ;)